rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(book, person, date)
+    @date = date
+    @book = book
+    book.rentals.push(self)
+    @person = person
+    person.rentals.push(self)
```

Comment:
- [ ] A person has many rentals, a method for adding rentals should be present in the `Person class`, something like so: 

```ruby
  def add_rental(book, date)
    # *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    Rental.new(date, book, self)
  end
```